src/model.ts

```diff
@@ -1,6 +1,18 @@
+export enum TerminalColor {
+  "black" = "terminal.ansiBlack",
+  "red" = "terminal.ansiRed",
+  "green" = "terminal.ansiGreen",
+  "yellow" = "terminal.ansiYellow",
+  "blue" = "terminal.ansiBlue",
+  "magenta" = "terminal.ansiMagenta",
+  "cyan" = "terminal.ansiCyan",
+  "white" = "terminal.ansiWhite",
+}
```

Comment:
Why put this in an enum if you're gonna use it like a map? Imo it's better to do a readonly object.